### ISSUE TITLE
Notifications should have aria-live labels

### ISSUE THREAD

**Issue by davidindub** [davidindub/coffeecrew#47]

**Comment by davidindub** [comment]
Made Messages/Notifications dismissible

**Comment by davidindub** [comment]
`aria-live` not appropriate as messages only appear on page load, `role="alert"` used instead.

Tested with VoiceOver on macOS.
